src/__init__.py

```diff
@@ -732,10 +732,11 @@ def get_pixmap(self, matrix=None, dpi=None, colorspace=None, alpha=0):
         if not cs:
             cs = mupdf.fz_device_rgb()
 
-        pix = mupdf.pdf_new_pixmap_from_annot( self.this, ctm, cs, mupdf.FzSeparations(0), alpha)
+        pix = mupdf.pdf_new_pixmap_from_annot(self.this, ctm, cs, mupdf.FzSeparations(0), alpha)
         if dpi:
-            pix.set_dpi(dpi, dpi)
-        return Pixmap( pix)
+            pix.m_internal.xres = dpi
+            pix.m_internal.yres = dpi
```

Comment:
Why this change - isn't it just duplicating the code in `Pixmap.set_dpi()`?

Reply:
Oh, i see the reference to #3414. But might it be better to do:
```
ret = Pixmap(pix)
ret.set_dpi(dpi, dpi)
return ret
```
? This way we make use of the existing `Pixmap.set_dpi()`, instead of duplicating it.

Reply:
Ah, ok - didn't know this would work like that.

Reply:
fixed